docs/types/value-types.rst

```diff
@@ -333,6 +333,10 @@ on ``call``.
     All contracts can be converted to ``address`` type, so it is possible to query the balance of the
     current contract using ``address(this).balance``.
```

Comment:
I actually meant moving it up above this one as well:-).
The first note "It is best to avoid..." still refers to the "* ``call``..." bullet point, but the last note is general and refers to address members in general, so I guess it should come after the new bullet point.

If you move it up once more and squash to a single commit, I'll merge!